pyproject.toml

```diff
@@ -29,7 +29,7 @@ adr-tools-python = "^1.0.3"
 black = "^23.3.0"
 httpx = "^0.24.0"
 isort = "^5.12.0"
-mkdocs = "^1.4.2"
+mkdocs = "^1.6.0"
```

Comment:
why is the mkdocs dependency in the dev dependencies section instead of the docs section? shouldn't it be moved?

Reply:
do not remember why, but makes sense to be only in docs

will make the change

Reply:
done